docs/docs/dev_docs/contracts/compiling.md

```diff
@@ -1,22 +1,136 @@
 # Compiling contracts
 
-Please use the [TUTORIAL-TEMPLATE](../../TUTORIAL_TEMPLATE.md) for standalone guides / tutorials.
```

Comment:
Nobody likes the template :(

Reply:
Heh, sorry! I felt it was too extensive for what I was looking for. Maybe having a slightly abridged version?

Reply:
I was taking inspiration from the uniswap developer guides. They all follow the exact same format, so there's no variance depending on the author of the file. Each page presents info in the same way. But no one's using the template. Maybe devrel can choose to iterate towards a template in future.